docs/gc.md

```diff
@@ -83,17 +76,17 @@ The *nursery* area is subdivided into two further areas, the *allocate* space an
 3. A local GC scavenge process runs and reachable objects are either copied into the *survivor* space or into the *tenure* area if they have reached *tenure age*. Any objects that can't be reached are left untouched and subsequently cleared.
 4. The *allocate* and *survivor* spaces swap roles. The original *survivor* space becomes the *allocate* space where new objects are created, and the original *allocate* space becomes the *survivor* space ready for the next local GC scavenge process.
 
-The relative sizes of the *allocate* and *survivor* spaces are dynamically adjusted by a technique called *tilting*. When the *nursery* area is first created, it is evenly divided between the *allocate* and *survivor* spaces. If, after a GC scavenge process is run, the amount of space required for the *survivor* area is comparatively small, the boundary between the two spaces is adjusted by *tilting*. For example, if the *survivor* space requires only 10% of the *nursery* area, the *tilt ratio* is adjusted to give 90% of the *nursery* area to the *allocate* space. With more space available for new objects, garbage collection can be delayed.
+The relative sizes of the *allocate* and *survivor* spaces are dynamically adjusted by a technique called *tilting*. When the *nursery* area is first created, it is evenly divided between the *allocate* and *survivor* spaces. If, after a GC *scavenge* operation runs, the amount of space required for the *survivor* area is comparatively small, the boundary between the two spaces is adjusted by *tilting*. For example, if the *survivor* space requires only 10% of the *nursery* area, the *tilt ratio* is adjusted to give 90% of the *nursery* area to the *allocate* space. With more space available for new objects, a global GC cycle can be delayed.
```

Comment:
Conclusion is not quite right: With more space available for new objects, a global GC cycle can be delayed.

With tilting, we are not distributing the space between tenure and nursery, but only within nursery. 

The right conclusion is:
With more space available to allocate (out of whole nursery), we decrease frequency of scavenge cycles.

The original conclusion is indirectly correct, but we would need 2 things to happen: both tilt to increase allocate within nursrey, and contract nursery (to reduce allocate to its pre-tilt size) to allow tenure to grow (and only if heap is already expanded) for how much nursery decreased. But I don't feel we need to elaborate indirect/potential consequences.

Reply:
Corrected: " With more space available for new objects, the frequency of *scavenge* cycles is reduced."